genesyscloud/integration_credential/genesyscloud_integration_credential_proxy.go

```diff
@@ -48,6 +49,7 @@ type integrationCredsProxy struct {
 	getIntegrationCredByNameAttr getIntegrationCredByNameFunc
 	updateIntegrationCredAttr    updateIntegrationCredFunc
 	deleteIntegrationCredAttr    deleteIntegrationCredFunc
+	getIntegrationByIdAttr       getIntegrationByIdFunc
```

Comment:
I think you forgot to give this a value? As in the actual implementation function that calls the API `getIntegrationByIdFn`

Reply:
Good catch, thank you. I've updated the code & tested successfully. I didn't see the error on my earlier test because i was not testing against the last version of my build. It looks like I am having a problem with the value resolved by var PLUGINS_DIR in the makefile, which points to a folder that is not the one terraform is looking at. I initially caught this error when I started setting things up and forgot I recently commented out my local fix. I will create a ticket for this to investigate further.